packages/libretro/core-info/package.mk

```diff
@@ -19,7 +19,7 @@
 ################################################################################
 
 PKG_NAME="core-info"
-PKG_VERSION="44a65bc"
+PKG_VERSION="4a020d4"
```

Comment:
That's wrong, bas copy paste from core package.mk

Reply:
fixed